src/dune_lang/value.ml

```diff
@@ -79,3 +79,40 @@ module L = struct
   let paths = List.map ~f:(fun x -> Path x)
   let dirs = List.map ~f:(fun x -> Dir x)
 end
+
+module Deferred_concat = struct
+  type value = t
+  type t = value list
+
+  let singleton = List.singleton
+
+  (* Helper to place a value [sep] between every value in a list *)
+  let rec list_intersperse xs ~sep =
```

Comment:
Could you move  this function to stdune?

Reply:
done